### PR TITLE
stolostron/backlog#26798 search deprec

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/deprecate_remove.adoc [hunk 1]
```diff
@@ -218,6 +218,18 @@ A _removed_ item is typically function that was deprecated in previous releases
...
+| `SearchCustomization.open-cluster-management.io` custom resource definition is removed
+| 2.7
+| Use custom resource definition `search.open-cluster-management.io/v1alpha1`. 
+| 
+
+| Search
+| RedisGraph is replaced as the source database
+| 2.7
+| No change required. 
+| PostgreSQL replaced RedisGraph as the source database.
```
Comment (jlpadilla) [inline release_notes/deprecate_remove.adoc]: Would this be better?
```suggestion
| Search
| PostgreSQL replaced RedisGraph as the internal database
| 2.7
| No change required. 
| The search component was reimplemented using PostgreSQL as the internal database.
```

File: release_notes/deprecate_remove.adoc [hunk 2]
```diff
@@ -218,6 +218,18 @@ A _removed_ item is typically function that was deprecated in previous releases
 |===
 |Product or category | Affected item | Version | Recommended action | More details and links
 
+| Search
+| `SearchCustomization.open-cluster-management.io` custom resource definition is removed.
```
Comment (jlpadilla) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| `SearchCustomizations.open-cluster-management.io` custom resource definition is removed.
```

File: release_notes/deprecate_remove.adoc [hunk 3]
```diff
@@ -218,6 +218,18 @@ A _removed_ item is typically function that was deprecated in previous releases
...
+| 2.7
+| Use custom resource definition `search.open-cluster-management.io/v1alpha1`.
```
Comment (jlpadilla) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| Use `search.open-cluster-management.io/v1alpha1` to customize search.
```

File: release_notes/deprecate_remove.adoc [hunk 4]
```diff
@@ -218,6 +218,18 @@ A _removed_ item is typically function that was deprecated in previous releases
...
 |Product or category | Affected item | Version | Recommended action | More details and links
 
+| Search
+| `SearchCustomization.open-cluster-management.io` custom resource definition is removed.
+| 2.7
+| Use custom resource definition `search.open-cluster-management.io/v1alpha1`. 
+| 
+
+| Search
+| PostgreSQL replaced RedisGraph as the internal database.
```
Comment (jlpadilla) [inline release_notes/deprecate_remove.adoc]:
```suggestion
| RedisGraph was replaced by PostgreSQL as the internal database.
```

**Review by jlpadilla — APPROVED** [review]
/lgtm

File: release_notes/deprecate_remove.adoc [hunk 5]
```diff
@@ -218,6 +218,18 @@ A _removed_ item is typically function that was deprecated in previous releases
...
+| `SearchCustomizations.open-cluster-management.io` custom resource definition is removed.
+| 2.7
+| Use `search.open-cluster-management.io/v1alpha1` to customize search.
+| 
+
+| Search
+| RedisGraph was replaced by PostgreSQL as the internal database.
+| 2.7
+| No change required. 
+| The search component was reimplemented using PostgreSQL as the internal database.
```
Comment (dockerymick) [inline release_notes/deprecate_remove.adoc]: I recommend that you change "was" to "is"

**Review by dockerymick — APPROVED** [review]
Left a suggestion, otherwise this looks good to me!

**Pull request merged**